sim.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import cv2
+import magic_numbers
+import numpy as np
+from camera_manager import MockImageManager, MockVideoManager, WebcamCameraManager
+from connection import DummyConnection
+from power_port_vision import Vision
+
+parser = argparse.ArgumentParser(
+    "Simulate code by giving it an image, list of images, video, or camera."
+)
+parser.add_argument("-v", "--video", help="Use a video")
+parser.add_argument(
+    "-i",
+    "--image",
+    nargs="+",
+    help="Specify an image or list of images. for example, `sim.py --image foo.jpg bar.jpg",
+)
+parser.add_argument(
+    "-c",
+    "--camera",
+    help="Use a camera. Must be followed by a camera number. For example, most webcams would be 0.",
+)
+parser.add_argument(
+    "-pp", "--power-port", help="Simulate with power port code", action="store_true"
```

Comment:
As opposed to what? If the loading bay is an option, it should be mentioned and used, or really this just shouldn't be here until we need it. In general, avoid the temptation to put anything in "for later". Look up YAGNI on the net. :-)